topi/python/topi/nn/l2_norm.py

```diff
@@ -0,0 +1,36 @@
+# pylint: disable=invalid-name
+"""TVM operator for l2norm"""
+from __future__ import absolute_import
+import tvm
+
+@tvm.target.generic_func
+def l2norm_instance_nchw(data, eps):
+    """Perform local response normalisation on the data
```

Comment:
We should write down the formula like in PyTorch (https://github.com/pytorch/pytorch/blob/master/torch/nn/modules/normalization.py#L9-L36) and TensorFlow (https://www.tensorflow.org/api_docs/python/tf/nn/local_response_normalization).